gemrb/GUIScripts/MessageWindow.py

```diff
@@ -40,8 +40,8 @@ def OnLoad():
 	TMessageTA.SetFlags(IE_GUI_TEXTAREA_AUTOSCROLL|IE_GUI_TEXTAREA_HISTORY)
 	TMessageTA.SetResizeFlags(IE_GUI_VIEW_RESIZE_ALL)
 	TMessageTA.AddAlias("MsgSys", 0)
-	TMessageTA.SetColor({'r' : 255, 'g' : 0, 'b' : 0}, TA_COLOR_OPTIONS)
-	TMessageTA.SetColor({'r' : 255, 'g' : 255, 'b' : 255}, TA_COLOR_HOVER)
+	TMessageTA.SetColor(ColorRed, TA_COLOR_OPTIONS)
+	TMessageTA.SetColor(ColorWhite, TA_COLOR_HOVER)
```

Comment:
I bet these should be the whitish kind.

Reply:
As with the other listbox colors I can't find reference for it in the CHU via NI. I am trusting your judgement here and changed it accordingly.

Reply:
You can always use a color picker for an estimate, but this was just going off of the fact that pure colors are rare.

Reply:
This is a good idea! But even with a color picker I get no better answers. The white values I get with a picker for the HOVER color are not unambiguous and span a rather wide range of values. In the end I resorted to plain old screenshots ;)

GemRB with `ColorWhitish`
![gemrb](https://github.com/gemrb/gemrb/assets/155195419/20d8fccd-958e-4b80-a277-0d5b98e3e6e0)

Original
![wine](https://github.com/gemrb/gemrb/assets/155195419/f8d8767d-a6ab-4421-b17a-f51907db4fc9)

GemRB with `ColorWhite`
![gemrb1](https://github.com/gemrb/gemrb/assets/155195419/816f94bf-a40f-4b44-8dba-23dc2929b21e)

I'd say `ColorWhite` is closer to the original than `ColorWhitish`. Without doing deeper research we probably won't get the correct original value now. I will revert the color back to `ColorWhite`. Getting the correct original colors is a topic for a differnt issue/PR.